tests/config/test_data.config

```diff
@@ -2,12 +2,20 @@
 params {
     // Default endpoint provided by nf-core, we keep it here in case sometimes we end up pushing stuff to them
     test_data_base = "https://github.com/scilus/nf-scil/raw"
+    test_data_base_n4 = "https://github.com/arnaudbore/nf-scil/raw"
 
     test_data {
         "scilpy" {
             "cropvolume" {
-                image = "${params.test_data_base}/main/.test_data/scilpy/crop_volume_image.nii.gz"
-                mask = "${params.test_data_base}/main/.test_data/scilpy/crop_volume_mask.nii.gz"
+                image = "${params.test_data_base}/main/.test_data/scilpy/dwi.nii.gz"
+                mask = "${params.test_data_base}/main/.test_data/scilpy/mask.nii.gz"
+            }
+        }
+        "preproc" {
+            "n4" {
+                dwi = "${params.test_data_base_n4}/n4_module/.test_data/preproc/dwi.nii.gz"
```

Comment:
You can switch back those endpoints to target the `main` branch on `scilus` nf-scil